### PR TITLE
(#4019) Fixes production logout destination

### LINKED ISSUE #4019: Logging out on PROD sends you to Okta Stage
### Issue description
Hitting the logout button on PROD or waiting for the session timeout sends you to the Okta staging environment rather than the production environment.

> **ESTIMATE** 

1

### Steps to reproduce the issue

1. Log in to Prod CMS 
2. Log out of PROD CMS
3. Expect to be on Okta Prod. Actually on Okta Stage.

### PR DESCRIPTION
**Pull request by jfrank-nih**
Closes #4019

### CODE REVIEW

File: docroot/sites/settings/cgov_core.settings.php [hunk 1]
```diff
@@ -63,12 +63,14 @@
 switch ($env) {
   case '01live':
```
Comment (bryanpizzillo) [inline docroot/sites/settings/cgov_core.settings.php:64]: Given this issue, can we move 01live below the dev blocks? Just that would have highlighted the missing break.

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]

File: docroot/sites/settings/cgov_core.settings.php [hunk 2]
```diff
@@ -61,15 +61,16 @@
...
-  case '01live':
-    $config['simplesamlphp_auth.settings']['logout_goto_url'] = 'https://iam.cancer.gov/login/signout';
   case '01dev':
   case 'dev':
   case '01test':
   case 'test':
   case 'int':
     $config['simplesamlphp_auth.settings']['logout_goto_url'] = 'https://iam-stage.cancer.gov/login/signout';
+    break;
+  // Production for default, including 01live
```
Comment (bryanpizzillo) [inline docroot/sites/settings/cgov_core.settings.php:71]: Can you please move the comment right above `$config`? It breaks up the switch readability.

**Review by blairlearn — APPROVED** [review]

**Comment by alinai** [comment]
@bryanpizzillo , @jfrank-nih ,tested and the test result was verified by @jfrank-nih. Thanks @jfrank-nih for confirming the test result!

**Review by bryanpizzillo — APPROVED** [review]

**Comment by SupriyaSaxena0605** [comment]
@bryanpizzillo @jfrank-nih Verified the log out page on CMS-dev.

**Pull request merged**